src/runtime/contrib/tflite/tflite_runtime.h

```diff
@@ -60,6 +60,9 @@ class TFLiteRuntime : public ModuleNode {
    */
   const char* type_key() const { return "TFLiteRuntime"; }
 
+  /*! \brief Get the property of the runtime module .*/
+  uint8_t GetProperty() const final { return property::kRunnable; };
```

Comment:
make the return value as int as that is more common in bitmasking

Reply:
Done